client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -271,11 +298,99 @@ const getPackagesErrors = values =>
 		return errors;
 	} );
 
-export const getCustomsErrors = customs => {
-	if ( ! customs.dummyFieldChecked && customs.submitted ) {
-		return { dummyField: 'This must be selected to continue' };
-	}
-	return {};
+export const getCustomsErrors = (
+	packages,
+	customs,
+	destinationCountryCode,
+	destinationCountryName
+) => {
+	const usedProductIds = uniq(
+		flatten( map( packages, pckg => map( pckg.items, 'product_id' ) ) )
+	);
+
+	const valuesByProductId = zipObject( usedProductIds, fill( Array( usedProductIds.length ), 0 ) );
+	forEach( packages, pckg => {
+		forEach(
+			pckg.items,
+			( { quantity, product_id } ) =>
+				( valuesByProductId[ product_id ] += quantity * customs.items[ product_id ].value )
```

Comment:
`customs.items[ product_id ].value` is undefined when using an older version of the plugin (before the change in https://github.com/Automattic/woocommerce-services/pull/1382). Should we make sure that users can't use customs / intl shipping with an older version of the plugin and/or when we're unable to retrieve the product values?

Reply:
Oooh! Even with international shipping enabled in the plugin, we may return `NULL` for the product price (the null return value is here: https://github.com/Automattic/woocommerce-services/pull/1382/files#diff-96cf36301ddb98bc2d8704fd75638444R157).

This is problematic because in this case, we don't know if a product being shipped has a value of more than $2500, and we don't alert the user.

It's also problematic because we're not using the correct price in the customs form.

We need to either always find the product price, or alert the user that we don't have it so they can add their own product value/price, or troubleshoot.

I ran into this problem right now with my test site, when running the international shipping support plugin code from https://github.com/Automattic/woocommerce-services/pull/1382/.

Reply:
> This is problematic because in this case, we don't know if a product being shipped has a value of more than $2500, and we don't alert the user.

I'm not too worried about this because that alert is a "best-effort" kind of thing. We can't cover any possible scenario where an ITN may be needed, we can just try our best.

> It's also problematic because we're not using the correct price in the customs form.

This, oin the other hand, worries me. In the first iteration of the Customs step, the item's price was displayed, so the merchant could see that it's `$0.00` and investigate, but not anymore.

I honestly don't know what's the best course of action here. I lost a lot of time making this change backwards-compatible with older versions of the plugin but I think it's better to just require a version of the plugin with the international labels changes.

Reply:
Sounds like not making it backwards compatible is the best solution here, and to use `wc/v3` right away.

Reply:
@DanReyLop just wanted to point out that we sometimes send `NULL` even if we're using the newest version of the plugin - (the null return value is here: https://github.com/Automattic/woocommerce-services/pull/1382/files#diff-96cf36301ddb98bc2d8704fd75638444R157).

So, it's not just about making it backwards compatible, but making sure that we're always sending a price down, or asking the merchant to enter a price if we get a `NULL` value from the server.

Reply:
> I'm not too worried about this because that alert is a "best-effort" kind of thing. We can't cover any possible scenario where an ITN may be needed, we can just try our best.

I disagree. Users are trusting us with an official form. If we can't validate it for them, we need to make sure we are at least pointing that they have to make sure to enter the right amounts themselves.

In other parts of the form, we do validate their data, which makes a user think other parts are validated, too. An example of this trust that we're building in other parts of the form - we'll make when the ITN is needed (e.g. if shipping to a specific country).

Moreover - we _can_ validate this for our users pretty easily, which makes for a much better experience, so we should do it.

Reply:
> Moreover - we can validate this for our users pretty easily, which makes for a much better experience, so we should do it.

We *can* validate this particular case, but we **can't** say with a 100% certainty that any given shipment won't need an ITN. [From the official documentation](https://pe.usps.com/text/imm/immc5_010.htm):

> ITN is required if [...]
The item requires an export license under U.S. law. (See 510, 530, 540, 560, and 590 for additional information.)

We can't know that in our validator, therefore the user shouldn't trust blindly that we will tell them if they need an ITN. How do you think we could communicate that better? @kellychoffman ?

Reply:
It would be great if we told the users things we could validate for them, and point out things we are not able to, with appropriate links to resources.

> We checked your customs form for these things:
> ✔the countries you're shipping to
> ✔the total value of each of the items you are shipping
> There are things we can't check for, for example:
> ⃝ shipping plants and other goods that require quarantine
> ⃝ item requires an export license under U.S. law

Etc